### DISCUSSION TITLE
Saucedacity 1.2 Is Almost Here!

### DISCUSSION THREAD

**Discussion by generic-pers0n** [saucedacity/saucedacity#39] [Announcements]
Alright, so Saucedacity 1.2 is very close to a release! I will be putting the final touches beginning now, mainly with our new default theme, etc.

As for the Tenacity-Saucedacity discussions, it appears that parts of Tenacity won't make it in time for our upcoming release. They will be saved for Saucedacity 1.3 instead and future releases. I am still looking forward to our Saucedacity 1.3 release.

If you guys have any last-minute thoughts, let me know now. I am currently expecting Saucedacity 1.2 to be released some time this month in August.

**Comment by Paul-Licameli** [comment]
Upstream says "you're welcome" https://github.com/saucedacity/saucedacity/commit/63c286ce66b5149dc3465499334f64fb6277f714

**Reply by generic-pers0n** [reply]
Yeah. You guys should receive some credit too, given that you had authored these commits...
